test/built-ins/Proxy/construct/trap-is-undefined-proto-from-ctor-realm.js

```diff
@@ -25,9 +25,10 @@ features: [cross-realm, Proxy, Reflect, Reflect.construct]
 
 var other = $262.createRealm().global;
 var C = new other.Function();
+C.prototype = null;
 
 var P = new Proxy(function() {}, {});
 
 var p = Reflect.construct(P, [], C);
 
-assert.sameValue(Object.getPrototypeOf(Object.getPrototypeOf(p)), other.Object.prototype);
+assert.sameValue(Object.getPrototypeOf(p), other.Object.prototype);
```

Comment:
the former formatting were passing on SpiderMonkey and the new version is not.

I'd suggest creating a new bug for FF to report it.

Meanwhile, can we have both versions in 2 separate files? I'd like to preserve the old test as well.

Reply:
021019b

Reply:
Test results recap (this file, new version):

[SpiderMonkey v67.0b19](https://circleci.com/gh/tc39/test262/1260): FAIL
[SpiderMonkey v68.0b3](https://circleci.com/gh/tc39/test262/1705): PASS

[ChakraCore v1.11.8](https://circleci.com/gh/tc39/test262/1261): FAIL
[ChakraCore v1.11.9](https://circleci.com/gh/tc39/test262/1706): FAIL